sphinx/quickstart.py

```diff
@@ -213,9 +213,10 @@
 # directly to the root of the documentation.
 #html_extra_path = []
 
-# If not '', a 'Last updated on:' timestamp is inserted at every page bottom,
-# using the given strftime format.
-#html_last_updated_fmt = '%%b %%d, %%Y'
+# If not None, a 'Last updated on:' timestamp is inserted at every page
+# bottom, using the given strftime format.
+# The empty string is equivalent to '%b %d, %Y'.
+#html_last_updated_fmt = None
```

Comment:
Oh, maybe we have different understanding of what "default" means. Note that this part in quickstart.py ends up in a newly generated conf.py that is given to the end user. So if nothing is changed in such a conf.py, the default values apply, right? What happens if html_last_updated_fmt appears with a # in conf.py? Right, the "Last updated" text will not be generated in the resulting HTML files. So the "default" is None and not '%b %d, %Y'.